### DISCUSSION TITLE
Adding jsx-a11y

### DISCUSSION THREAD

**Discussion by KasperAndersson** [NoA-Ignite-dk/eslint-config#5] [Ideas]
Add https://github.com/jsx-eslint/eslint-plugin-jsx-a11y

**Comment by jdpnielsen** [comment]
@KasperAndersson Which rules would you want from this package? Maybe you could make a PR to get the discussion going.

**Reply by KasperAndersson** [reply]
All of them i guess, in theory this should be best practice. I'll open a PR.
